tools/cldr-code/src/main/java/org/unicode/cldr/util/ApproximateWidth.java

```diff
@@ -25,22 +23,22 @@ public static int getWidth(CharSequence s) {
     }
 
     static {
-        SemiFileReader MyFileHander = new SemiFileReader() {
-            @Override
-            public void handleComment(String line, int commentCharPosition) {
-                if (line.contains("@missing")) {
-                    String[] items = SPLIT.split(line);
-                    defaultWidth = Integer.parseInt(items[1]);
-                }
-            }
-
-            @Override
-            protected boolean handleLine(int lineCount, int start, int end, String[] items) {
-                data.putAll(start, end, Integer.parseInt(items[1]));
-                return true;
-            }
-
-        };
+        SemiFileReader MyFileHander =
+                new SemiFileReader() {
```

Comment:
Extra indentation is because this is a wrapped line, and to distinguish wrapped lines from following statements, they get 2 x indentation.

See also:
https://docs.google.com/document/d/1JH4KH3jNMFhEYFb8G85adOMgn98iJtSHqs6jNt21-rs/edit#heading=h.9bojtti372ih

This is going to be by far the biggest change in lines of code if we switch to the Google formatter.